src/Microsoft.Diagnostics.Runtime/src/Builders/RuntimeBuilder.cs

```diff
@@ -109,15 +110,19 @@ public void Dispose()
 
 
 
-        bool IHeapHelpers.CreateSegments(ClrHeap clrHeap, out IReadOnlyList<ClrSegment> segments, out IReadOnlyList<AllocationContext> allocationContexts,
-                        out IReadOnlyList<FinalizerQueueSegment> fqRoots, out IReadOnlyList<FinalizerQueueSegment> fqObjects)
+        bool IHeapHelpers.CreateSegments(
+            ClrHeap clrHeap,
+            out IReadOnlyList<ClrSegment> segments,
+            out IReadOnlyList<AllocationContext> allocationContexts,
+            out IReadOnlyList<FinalizerQueueSegment> fqRoots,
+            out IReadOnlyList<FinalizerQueueSegment> fqObjects)
         {
             List<ClrSegment> segs = new List<ClrSegment>();
             List<AllocationContext> allocContexts = new List<AllocationContext>();
             List<FinalizerQueueSegment> finalizerRoots = new List<FinalizerQueueSegment>();
             List<FinalizerQueueSegment> finalizerObjects = new List<FinalizerQueueSegment>();
 
-            if (allocContexts.Count == 0)
+            if (true)
```

Comment:
Ah, the `allocContexts.Count == 0` is bad leftover code from when I tried to cache that value earlier.  Can you just remove the `if (true)`?

Reply:
and the brackets